src/components/OperationsList/index.js

```diff
@@ -91,89 +78,124 @@ const ShowMore = styled(Box).attrs({
   p: 4,
   color: 'wallet',
 })`
-  border-top: 1px solid ${p => p.theme.colors.fog};
   cursor: pointer;
   &:hover {
     text-decoration: underline;
   }
 `
 
+const AddressEllipsis = styled.div`
+  display: block;
+  flex-shrink: 1;
+  min-width: 20px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`
+
+const Address = ({ value }: { value: string }) => {
+  const addrSize = value.length / 2
+
+  const left = value.slice(0, 10)
+  const right = value.slice(-addrSize)
+  const middle = value.slice(10, -addrSize)
+
+  return (
+    <Box horizontal color="smoke" ff="Open Sans" fontSize={3}>
```

Comment:
should `Open Sans` be in a constant? same Q for `smoke` 🤔

Reply:
Actually, it's a constant :D
See: https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/styles/theme.js

Reply:
holy smoke :D 👍